docs/developer-guide/checks.md

```diff
@@ -1,370 +1,328 @@
-# Create a new Check for a Provider
+# Prowler Checks
 
-Here you can find how to create new checks for Prowler.
-
-**To create a check is required to have a Prowler provider service already created, so if the service is not present or the attribute you want to audit is not retrieved by the service, please refer to the [Service](./services.md) documentation.**
+This guide explains how to create new checks in Prowler.
 
 ## Introduction
 
-The checks are the fundamental piece of Prowler. A check is a simply piece of code that ensures if something is configured against cybersecurity best practices. Then the check generates a finding with the result and includes the check's metadata to give the user more contextual information about the result, the risk and how to remediate it.
+Checks are the core component of Prowler. A check is a simple piece of code designed to validate whether a configuration aligns with cybersecurity best practices. Execution of a check yields a finding, which includes the result and contextual metadata (e.g., outcome, risks, remediation).
```

Comment:
I like to avoid words like "simple" or "easy" whenever possible in technical guides as these words make assumptions about the reader’s knowledge.

Also, some checks are "simple" (meaning that maybe they check directly if some setting is true/false) but others are more "complex" and verify very specific setting or situations.